pkg/graph/resolvers/added_as_collaborator_email_test.go

```diff
@@ -62,13 +62,14 @@ func (s *ResolverSuite) TestAddedAsCollaboratorEmail() {
 
 	_, _, err := CreatePlanCollaborator(
 		context.Background(),
+		s.testConfigs.Store,
+		s.testConfigs.Store,
```

Comment:
Like above, why are we passing the store twice?

Reply:
yes, see open conversation